recipes-multimedia/gstreamer/gstreamer1.0-python_1.26.0.bbappend

```diff
@@ -0,0 +1 @@
+DEPENDS:append = " gstreamer1.0-plugins-bad"
```

Comment:
You need to create the copy of the recipe here with the suffix, so we have a consistent version, even if doing the backport in the future.

Reply:
We have been adding the suffix to ensure proper identification of the recipe, which is not sourced from the project's official repositories. That is why we have been using this approach.